### ISSUE TITLE
Upgrade manifest.json to Manifest version v3

### ISSUE THREAD

**Issue by Patricek0056** [lightbeam24/customtube#15]
### Problem description:
Hello, I want to say, that the Manifest v2 is slowly being outdated, so it would be good if you update this extension to Manifest v3.

**Comment by lightbeam24** [comment]
I plan to. I've decided that I'm going to build the Manifest V3 version of the extension around Firefox 115 when it releases, as that is the version of Firefox that Windows 7 and 8.1 users will be stuck on (they'll be moved to 115 ESR). That is also when I plan to try to port it to Chromium.

**Comment by lightbeam24** [comment]
No, absolutely not! Right now it's exclusive to Firefox, partly due to the fact that it's Manifest V2. Once it is Manifest V3, I can support both Chromium *and* Firefox.

**Comment by Patricek0056** [comment]
Sorry for misunderstanding and confusion. I just understood it wrong, sorry.

**Comment by Patricek0056** [comment]
But this issue will be opened until upgrade to Mv3, I'm right? Or you will close this for now?

**Comment by lightbeam24** [comment]
I'll leave it open until the issue is resolved.

**Comment by lightbeam24** [comment]
Transition to MV3 on the Firefox side has been postponed, as Mozilla themselves do not recommend using it yet for compatibility reasons. However, the transition has been done for the newly released Chrome version, and the Firefox betas are also Manifest V3.
